docs/releases/v0.21.0.md

```diff
@@ -0,0 +1,26 @@
+# Changelog
+
+## [v0.21.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.21.0) (2022-08-24)
+
+[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.20.0...v0.21.0)
+
+**Upstream release notes:**
+
+- [v0.58.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.58.0)
+- [v0.57.2](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.57.2)
+
+**Implemented enhancements:**
+
+- Export OTEL traces with Auth extension [\#1056](https://github.com/aws-observability/aws-otel-collector/issues/1056)
+
+**Merged pull requests:**
+
+- Update fargate deployment template to use include in cumulativetodelta processor [\#1435](https://github.com/aws-observability/aws-otel-collector/pull/1435) ([vasireddy99](https://github.com/vasireddy99))
+- Modify mirroring tool to use `crane` go pkg [\#1417](https://github.com/aws-observability/aws-otel-collector/pull/1417) ([erichsueh3](https://github.com/erichsueh3))
+- Update upstream collector dependancies to v0.58.0 [\#1414](https://github.com/aws-observability/aws-otel-collector/pull/1414) ([vasireddy99](https://github.com/vasireddy99))
+- Upgrade ADOT Collector version to v0.20.0 [\#1385](https://github.com/aws-observability/aws-otel-collector/pull/1385) ([Kausik-A](https://github.com/Kausik-A))
+- Add operator arm64 tests [\#1365](https://github.com/aws-observability/aws-otel-collector/pull/1365) ([erichsueh3](https://github.com/erichsueh3))
```

Comment:
I'm not sure any of these are relevant to the end-user other than perhaps #1414.